libc/utils/HdrGen/PublicAPICommand.cpp

```diff
@@ -102,6 +102,12 @@ void writeAPIFromIndex(APIIndexer &G,
     llvm::Record *RetValSpec = FunctionSpec->getValueAsDef("Return");
     llvm::Record *ReturnType = RetValSpec->getValueAsDef("ReturnType");
 
+    bool Guarded =
```

Comment:
nit: Add a comment that ideally we would group functions based on their guarding macro.

Reply:
Done.